### PR TITLE
Docs: 2.19 changelog

### PR DESCRIPTION
**Pull request by lukstbit**
Looked through the PRs since the last major release and tried to produce a changelog for 2.19.
Feel free to dismiss completely or propose changes/additions.

You can find the commits references that I filtered [here](https://gist.github.com/lukstbit/a3f6f45455c58ba902639bfd60310557).

### CODE REVIEW

File: changelog.asc [hunk 1]
```diff
@@ -2,6 +2,45 @@
...
+*** Drag and drop support on some parts of the application
+** Better support for cards flags management
+** SHA256 & SHA1 application certificate fingerprints can be viewed in repository README
+** Update audio focus with support for Android 15
+** Reduced application size
+** Improved accessibility
+*** Better support for larger text size
+*** Added info text for more UI elements
+** Allow native deck selection in statistics screen
+** Study options screen shows bury counts as desktop
```
Comment (mikehardy) [inline changelog.asc:26]:
```suggestion
** Study options screen shows same bury counts as desktop
```

File: changelog.asc [hunk 2]
```diff
@@ -2,6 +2,45 @@
...
+** App shows info about WebView version requirements at start
```
Comment (mikehardy) [inline changelog.asc:27]: I think this is only relevant to developers since we are trying to reproduce on emulators that don't receive WebView updates, everyone else gets updates over the air

```suggestion
```

File: changelog.asc [hunk 3]
```diff
@@ -2,6 +2,45 @@
...
+** Fixed several issues with the interaction between note editor and template editor
+** Fixed learn ahead limit not being applied after setting change
+** Fixed several issues related to statistics page
+** Fixed silent crash on startup due to fatal error
+** Fixed several issues related to TTS replay
+** Fixed several issues related to portrait/landscape screen orientation
+** Fixed multiple UI inconsistencies
+** Fixed several issues to the timebox in review screen
+** Fixed several issues around the previewer
+** Many other smaller fixes
```
Comment (mikehardy) [inline changelog.asc:42]: I do not feel strongly about this but I try to include humor from time to time when possible
I think most apps and most programming is too corporate and impersonal and it is just no fun

I think users like fun

Feel free to be like "no, seriously dude, it's not even funny" and remove it. Or add your own humor :-)

```suggestion
** Many other smaller fixes (still reading this? Congratulations, you made it to the end!)
```

**Review by mikehardy — CHANGES_REQUESTED** [review]
2 tiny things that are corrections
1 thing which is a completely deletable or debatable addition

**Comment by mikehardy** [comment]
This is a fantastic start, I think it's somewhere between 1 and 2 minutes from being "merge and go..." personally

File: changelog.asc (hunk 3 above)
Comment (lukstbit) [reply changelog.asc:42]: Love it!

File: changelog.asc [hunk 4]
```diff
@@ -2,6 +2,44 @@
 :sectanchors:
 = AnkiDroid Changelog
 
+== Version 2.19.0 (20241025)
+* Incorporating the work from three Gsoc projects and the work done in the last couple of months, we are proud to present the latest iteration of the best Android app for learning!  
+* We really appreciate the https://opencollective.com/ankidroid[donations], they paid for these features 🤝
+* Features
+** Anki upstream version 24.06.3 included
```
Comment (david-allison) [inline changelog.asc:9]: + FSRS version

File: changelog.asc [hunk 5]
```diff
@@ -2,6 +2,44 @@
...
+** Two new widgets for easier access to your decks and cards
+** Streamlined process of adding cloze notes to AnkiDroid through an Instant Add Note Editor
+** Improved the process of adding multimedia to your notes
+** Improved experience for devices with larger screens
+*** Better utilization of screen space
+*** Better mouse support
+*** Better keyboard shortcuts support
+*** Drag and drop support on some parts of the application
+** Better support for cards flags management
+** SHA256 & SHA1 application certificate fingerprints can be viewed in repository README
```
Comment (david-allison) [inline changelog.asc:19]: would remove

File: changelog.asc [hunk 6]
```diff
@@ -2,6 +2,44 @@
...
 = AnkiDroid Changelog
 
+== Version 2.19.0 (20241025)
+* Incorporating the work from three Gsoc projects and the work done in the last couple of months, we are proud to present the latest iteration of the best Android app for learning!  
+* We really appreciate the https://opencollective.com/ankidroid[donations], they paid for these features 🤝
+* Features
+** Anki upstream version 24.06.3 included
+** Two new widgets for easier access to your decks and cards
+** Streamlined process of adding cloze notes to AnkiDroid through an Instant Add Note Editor
+** Improved the process of adding multimedia to your notes
```
Comment (david-allison) [inline changelog.asc:12]: SVG previews

File: changelog.asc [hunk 7]
```diff
@@ -2,6 +2,44 @@
...
+** Improved accessibility
+*** Better support for larger text size
+*** Added info text for more UI elements
+** Allow native deck selection in statistics screen
+** Study options screen shows same bury counts as desktop
+** Latest translations from our community of https://crowdin.com/project/ankidroid[translators] (thank you!)
+* Fixes
+** Fixed several issues related to keyboard handling
+** Fixed out of memory errors when displaying many tags 
+** Fixed crash for tags with unicode characters
```
Comment (david-allison) [inline changelog.asc:31]: We did? This should be in 2.18.2 and .3

**Review by david-allison — COMMENTED** [review]
Changes to consider:

* MathJax update is a big one
* JS API: "addTagToNote"
* Suggestion to rename decks for ordering: "chapter 1" -> "chapter 01"
* Better handling of outdated webview
* Mic toolbar is now persistent in the Reviewer
* AnkiWeb icon (probably not for the changelog)
* Fix shared decks rate limit
* Hold to record audio
* We added renaming flags in 2.19

File: changelog.asc (hunk 7 above)
Comment (lukstbit) [reply changelog.asc:31]: You're right, it was cherry picked!

File: changelog.asc (hunk 4 above)
Comment (lukstbit) [reply changelog.asc:9]: I'll add only the version. The changelog has other  versions declared and someone going there might say that we are way behind.

**Comment by lukstbit** [comment]
I've edited your message above with what I ended up implementing(almost all of them).

File: changelog.asc [hunk 8]
```diff
@@ -2,6 +2,51 @@
 :sectanchors:
 = AnkiDroid Changelog
 
+== Version 2.19.0 (20241026)
+* Incorporating the work from three Gsoc projects and the work done in the last couple of months, we are proud to present the latest iteration of the best Android app for learning!  
+* We really appreciate the https://opencollective.com/ankidroid[donations], they paid for these features 🤝
+* Features
+** Includes Anki upstream version 24.06.3 with FSRS 0.6.4
```
Comment (david-allison) [inline changelog.asc:9]: Could we put the user facing version in brackets here (lifted from the Anki Chsngelog)

File: changelog.asc [hunk 9]
```diff
@@ -2,6 +2,51 @@
 :sectanchors:
 = AnkiDroid Changelog
 
+== Version 2.19.0 (20241026)
+* Incorporating the work from three Gsoc projects and the work done in the last couple of months, we are proud to present the latest iteration of the best Android app for learning!  
+* We really appreciate the https://opencollective.com/ankidroid[donations], they paid for these features 🤝
+* Features
+** Includes Anki upstream version 24.06.3 with FSRS 0.6.4 
+** Use backend mathjax and jquery libraries
```
Comment (david-allison) [inline changelog.asc:10]: I'd just say updated here. Needs capitalisation

File: changelog.asc [hunk 10]
```diff
@@ -2,6 +2,51 @@
...
+*** Better mouse support
+*** Better keyboard shortcuts support
+*** Drag and drop support on some parts of the application
+** Better support for cards flags management
+** Update audio focus with support for Android 15
+** Reduced application size
+** Improved accessibility
+*** Better support for larger text size
+*** Added info text for more UI elements
+** Render SVGs using webview
```
Comment (david-allison) [inline changelog.asc:25]: A new preview for SVG images

File: changelog.asc [hunk 11]
```diff
@@ -2,6 +2,51 @@
...
+** Allow native deck selection in statistics screen
```
Comment (david-allison) [inline changelog.asc:26]: Improved deck selection

**Review by david-allison — COMMENTED** [review]
Treat this as an approve. Nitpicks only. Nothing blocking

File: changelog.asc (hunk 8 above)
Comment (lukstbit) [reply changelog.asc:9]: I hope I understood this correctly.

**Review by david-allison — APPROVED** [review]

File: changelog.asc [hunk 12]
```diff
@@ -2,6 +2,51 @@
 :sectanchors:
 = AnkiDroid Changelog
 
+== Version 2.19.0 (20241026)
+* Incorporating the work from three Gsoc projects and the work done in the last couple of months, we are proud to present the latest iteration of the best Android app for learning!
```
Comment (Arthur-Milchior) [inline changelog.asc:6]: Most user probably don't know what GSoC is (and it's not the proper casing).

Also, @criticalAY did two projects. So it's strange to count the number of projects.

I'd rephrase as:
> Incorporating the work done in the last couple of months, including three contributors financed by the "Google Summer of Code" program, we are proud...

File: changelog.asc [hunk 13]
```diff
@@ -2,6 +2,51 @@
...
+== Version 2.19.0 (20241026)
+* Incorporating the work from three Gsoc projects and the work done in the last couple of months, we are proud to present the latest iteration of the best Android app for learning!  
+* We really appreciate the https://opencollective.com/ankidroid[donations], they paid for these features 🤝
+* Features
+** Includes Anki upstream version 24.06.3 with FSRS(4.5) 0.6.4
+** Updated backend MathJax and jQuery libraries
+** Two new widgets for easier access to your decks and cards
+** Streamlined process of adding cloze notes to AnkiDroid through an Instant Add Note Editor
+** Improved the process of adding multimedia to your notes
+** Improved experience for devices with larger screens
```
Comment (Arthur-Milchior) [inline changelog.asc:14]: I'd add "such as chromebook and tablet". Some people may learn that we can be used on chromebook too reading this word, maybe

File: changelog.asc [hunk 14]
```diff
@@ -2,6 +2,51 @@
 :sectanchors:
 = AnkiDroid Changelog
 
+== Version 2.19.0 (20241026)
+* Incorporating the work from three Gsoc projects and the work done in the last couple of months, we are proud to present the latest iteration of the best Android app for learning!  
+* We really appreciate the https://opencollective.com/ankidroid[donations], they paid for these features 🤝
+* Features
+** Includes Anki upstream version 24.06.3 with FSRS(4.5) 0.6.4
```
Comment (Arthur-Milchior) [inline changelog.asc:9]: You may want to add a link to https://github.com/open-spaced-repetition/fsrs4anki

**Review by Arthur-Milchior — COMMENTED** [review]
Thanks so much for doing it!

**Comment by lukstbit** [comment]
Review comments addressed, thanks Arthur.

**Comment by david-allison** [comment]
@mikehardy awaiting final approval I feel 🚀🚀

**Review by mikehardy — APPROVED** [review]

**Comment by mikehardy** [comment]
Fantastic! Thanks @lukstbit for driving it and everyone for polishing, looks great
